hieroplatform/src/main/java/org/hiero/sketch/table/HashSubSchema.java

```diff
@@ -49,4 +52,9 @@ public boolean isColumnPresent(final String name) {
     }
 
     public int getColumnCount() { return this.colNames.size(); }
+
+    @Override
+    public Iterator<String> iterator() {
```

Comment:
I got rid of this for now. The right order should come from the schema that it is used on top of.